pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -62,33 +83,86 @@ func (op *overcommitPlugin) Name() string {
 }
 
 /*
-User should give overcommit-factor through overcommit plugin arguments as format below:
+User should give overcommit factors (cpu-overcommit-factor, mem-overcommit-factor, other-overcommit-factor, overcommit-factor)
+through overcommit plugin arguments as format below:
+
+Here we give five examples:
+
+Example 1:
+Specify all three values: cpu-overcommit-factor, mem-overcommit-factor, and other-overcommit-factor simultaneously.
 
 actions: "enqueue, allocate, backfill"
 tiers:
 - plugins:
   - name: overcommit
     arguments:
+    cpu-overcommit-factor: 1.2
+    mem-overcommit-factor: 1.0
+    other-overcommit-factor: 1.2
+
+Example 2:
+Specifying only the overcommit-factor implies that all three factors are the same.
```

Comment:
Full Docs dont need to be put here. Just an example.

Reply:
done